RELEASE.md

```diff
@@ -0,0 +1,51 @@
+# Release process
+
+## To release a new version of **qtsass** on PyPI:
+
+* Ensure you have the latest version from upstream and update your fork
+
+      git pull upstream master
+      git push origin master
+
+* Clean the repo
+
+      git clean -xfdi
+
+* Update CHANGELOG.md
+
+* Update version in `__init__.py` (set release version, remove 'dev0')
+
+* Commit changes
+
+      git add .
+      git commit -m "Set release version"
+
+* Create distributions
+
+      python setup.py sdist bdist_wheel
+
+* Upload distributions
+
+      twine upload dist/* -u <username> -p <password>
+
+* Add release tag
+
+      git tag -a vX.X.X -m 'Release version'
+
+* Update `__init__.py` (add 'dev0' and increment minor)
+
+* Commint changes
+
+      git add .
+      git commit -m "Restore dev version"
```

Comment:
We usually write this message

    git commit -m "Back to work"

but it's up to you if you want to change it or not.